WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -588,6 +588,11 @@ def __setFlavour( self ):
       self.pp.flavour = 'SSHLSF'
       pilotRef = 'sshlsf://' + self.pp.ceName + '/' + os.environ['LSB_BATCH_JID']
 
+    #  SLURM batch system
+    if os.environ.has_key( 'SLURM_JOBID' ):
```

Comment:
Thanks. There are many other "os.environ.has_key(**) in pilotCommands.py.
I replaced all of them with "** in os.envirion"